src/utils/internal/getCallFrame.test.ts

```diff
@@ -168,3 +168,15 @@ test('Safari on macOS error stack', () => {
     './test/msw-api/setup-worker/printHandlers.mocks.ts@http://localhost:56460/main.js:1376:5',
   )
 })
+
+test('IE11 undefined or null error stack', () => {
```

Comment:
Could you please rename this test title to something like

```suggestion
test('handles the undefined stack trace', () => {
```